src/components/WalletConnect.jsx

```diff
@@ -94,6 +94,7 @@ export default function WalletConnect() {
                         const user = instantiateUser(owner_address);
                         addUserToFireStore(user, owner_address);
                       }
+                      navigate(window.location.href) // refresh on connect so balances and renderings are updated
```

Comment:
When I disconnect and reconnect my wallet, I am auto-redirected to the wrong page. I think this is the issue.

Reply:
By wrong page I mean: localhost:3000/http:localhost:3000